README.md

```diff
@@ -1,32 +1,576 @@
+  <?xml version="1.0" encoding="utf-8"?>
+  <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="YOUR_PACKAGE_NAME" android:versionCode="1" android:versionName="1.0" >
+   <uses-sdk android:minSdkVersion="10" android:targetSdkVersion="10" />
+    ...
+    <!-- Add this to your AndroidManifest.xml -->
+    <uses-permission android:name="android.permission.GET_ACCOUNTS" />
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE" />
+    <permission android:name="YOUR_PACKAGE_NAME.permission.C2D_MESSAGE" android:protectionLevel="signature" />
+    <uses-permission android:name="YOUR_PACKAGE_NAME.permission.C2D_MESSAGE" />
+    <!-- End of changes -->
+
+    <application android:allowBackup="true" android:icon="@drawable/ic_launcher" android:label="@string/app_name" android:theme="@style/AppTheme" >
+
+      <!-- Add this to your AndroidManifest.xml -->
+      <!-- Specifies the Google Play Services version that the app was compiled with -->
+      <meta-data android:name="com.google.android.gms.version" android:value="@integer/google_play_services_version" />
+      
+      <!-- Configure the aspect of the UI -->
+      <meta-data android:name="SWRVE_PUSH_ICON" android:resource="@drawable/ic_launcher"/>
+      <meta-data android:name="SWRVE_PUSH_ICON_MATERIAL" android:resource="@drawable/ic_launcher_material"/>
+      <meta-data android:name="SWRVE_PUSH_ICON_LARGE" android:resource="@drawable/ic_launcher_large" />
+      <meta-data android:name="SWRVE_PUSH_ACCENT_COLOR" android:resource="@android:color/black" />
+      <meta-data android:name="SWRVE_PUSH_ACTIVITY" android:value=".MainActivity"/>
+      <meta-data android:name="SWRVE_PUSH_TITLE" android:value="Your app title"/>
+
+      <receiver android:name="com.google.android.gms.gcm.GcmReceiver" android:exported="true" android:permission="com.google.android.c2dm.permission.SEND" >
+        <intent-filter>
+          <action android:name="com.google.android.c2dm.intent.RECEIVE" />
+          <action android:name="com.google.android.c2dm.intent.REGISTRATION" />
+          <category android:name="YOUR_PACKAGE_NAME" />
+        </intent-filter>
+      </receiver>
+      <service android:name="com.swrve.sdk.gcm.SwrveGcmIntentService" >
+        <intent-filter>
+          <action android:name="com.google.android.c2dm.intent.RECEIVE" />
+        </intent-filter>
+      </service>
+      <service android:name="com.swrve.sdk.gcm.SwrveGcmInstanceIDListenerService" android:exported="false" >
+        <intent-filter>
+          <action android:name="com.google.android.gms.iid.InstanceID" />
+        </intent-filter>
+      </service>
+      <!-- End of changes -->
+      ...
+   </application>
+  </manifest>
+  ```
+
+  Note that you must replace `YOUR_PACKAGE_NAME` with the main package of your app. The listener services for push notifications need to live in one of your app’s packages.
```

Comment:
probably add a little mention for "Your app title" from the code above, in case anyone who adds this is particularly lazy about copy-pasting the code